app/pages/project/access/ProjectAccessPage.tsx

```diff
@@ -196,6 +198,12 @@ export function ProjectAccessPage() {
     <>
       <PageHeader>
         <PageTitle icon={<Access24Icon />}>Access</PageTitle>
+        <DocsPopover
+          heading="Access"
+          icon={<Access16Icon />}
+          summary="Identity and access management roles determine who can view, edit, or administer this project. Silo roles are inherited from the organization, while project roles are specific to this project."
```

Comment:
"organization" made me flinch. Guess I am traumatized. https://github.com/oxidecomputer/omicron/pull/2626

I do like the first sentence, so it might be nice to add that to the silo one too. I'd get rid of "Identity and access management" and just say 

> Roles determine who can view, edit, or administer this project.

And on silo access you could do

> Roles determine who can view, edit, or administer this silo and the projects within it.

The second sentence is pretty good, but I'd get rid of organization like I said. I also think it would be worth mentioning what happens when you have a role from each, even though it makes the text longer, because that is so fundamental.

> Silo roles are inherited from the silo. If a user or group has both a silo and project role, the stronger role takes precedence.

Reply:
Nice.